.github/ISSUE_TEMPLATE/py_index_request.yaml

```diff
@@ -0,0 +1,32 @@
+---
+name: "User support: New Python package index request"
+description: "Template for users requesting new Python package index on Operate First Pulp instance."
+title: "[Pulp] New Python package index request:"
+labels: [pulp, user-support]
+assignees:
```

Comment:
@fridex do any of the users in this list need to be added to the operate-first org for assignments to work?

Reply:
I'm not sure about that. I can remove the ones that are not present in this org as a starting point for the template.

Reply:
@4n4nd adjusted, PTAL 👍🏻